src/app/AppRegistry.tsx

```diff
@@ -488,9 +489,15 @@ export const modules = defineModules({
     fullBleed: true,
     hidesBackButton: true,
   }),
-  Home: reactModule(HomeContainer, {
-    isRootViewForTabName: "home",
-  }),
+  Home: reactModule(
+    HomeContainer,
+    {
+      isRootViewForTabName: "home",
+    },
+    ArtsyNativeModule.isBetaOrDev && !unsafe_getFeatureFlag("ARPreferLegacyHomeScreen")
+      ? [homeViewScreenQuery]
+      : undefined
```

Comment:
I think we can just add the new home view query here because it's not used anywhere else.

```suggestion
    [homeViewScreenQuery]
```

Reply:
True - I wanted this as a safe guard just in case we would accidentally call the prefetch home query somewhere and we would end up exposing a query with an interface that is still changing  and break clients. Maybe I went too far here since I am also checking before calling `prefetchURL("/")`

Reply:
Yeah, both should be fine, and we can clean this up when we remove the feature flag. 👍